dashboard/models.py

```diff
@@ -300,6 +300,7 @@ class PositionChoices(models.TextChoices):
 
     title = models.CharField(max_length=45, choices=PositionChoices.choices, unique=True, blank=False)
 
+    @property
```

Comment:
it shouldn't because the part of the do_verify code that references this is:
if pos == 'ec' and Position.objects.get(brothers__user=user).in_ec:

which will only evaluate to true if 'ec' is in the decorator. In fact now that I think about it, this function is actually just an alternative way to do what we were doing with EC_POSITIONS and COMMITTEE_CHAIRS and EVENT_CHAIRS it's just really poorly documented how it actually works. In fact, you could replace the Position.objects.get(brothers_user=user).in_ec with EC_POSITIONS and it would accomplish the exact same purpose.

Which do you think makes more sense in terms of proper implementation?

Reply:
Well at the very least, these changes are outside the scope of this merge request, so dont worry about making those changes before we merge (might want to make an issue?).

I now understand what the `pos == 'ec'` is for based on your explanation.  I think that actually makes sense so that we can allow all EC members to have access to some pages but not others.  I think that changing it to use the explicitly defined versions would be best and not having the secret 'ec' tag you can pass in that no one knows about.  Should be able to read the function decorator and understand who has access.